corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -203,4 +199,28 @@ public int dropTable(String namespace, String tablename) {
         log.info("\n======================\n");
         return tableSize;
     }
+
+    /**
+     * Loads the table with random data
+     * @param namespace - the namespace where the table belongs
+     * @param tablename - table name without the namespace
+     * @return - number of entries in the table
+     */
+    public int loadTable(String namespace, String tablename) {
```

Comment:
can we add a comment explaining what the purpose of this operation is and when it should be used?

Reply:
sure